code/game/objects/machinery/mortar.dm

```diff
@@ -210,6 +210,13 @@
 		flick(icon_state + "_fire", src)
 		mortar_shell.forceMove(src)
 
+		if(istype(src, /obj/machinery/deployable/mortar/howitzer))
```

Comment:
Istype src is bad

Reply:
Two solutions : 
Either you override the proc for howitzer

Else, you add a var on the projectile or mortar to specify on which tally it should be added